person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] @Tafloninno great work adding the add_rental method, however, this method is supposed to take three parameters `date, self, book` (self being the person itself), this will help to achieve the 'has-many' association.